### PR TITLE
fix: fixed an error in the vue component editor(https://github.com/wi…

### PR DESCRIPTION
**Pull request by phk422**
…thastro/language-tools/issues/758)

## Changes

- Modify the regular expression for matching `defineProps`
- Removes the condition where comments exist when working with code
- Fixed an issue with 'defineProps' import error

### CODE REVIEW

**Comment by Princesseuh** [comment]
Awesome, thank you! Would it be possible to add tests? Essentially, add a new `x.test.js` file in the `test` folder of the package, import `toTSX` and make sure the code generated is valid.

File: packages/integrations/vue/package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "@types/chai": "^4.3.10",
     "astro": "workspace:*",
     "astro-scripts": "workspace:*",
+    "chai": "^4.3.10",
```
Comment (ematipico) [inline packages/integrations/vue/package.json:52]: We don't use `chai` anymore, you'll have to use `node:assert/strict`

**Review by ematipico — CHANGES_REQUESTED** [review]
The new tests use an assertion library that we don't use anymore.

File: pnpm-lock.yaml
Comment (ematipico) [inline pnpm-lock.yaml:1]: You have to revert this file too @phk422

**Pull request closed without merging**